vad_transcriber/audioTranscript_gui.py

```diff
@@ -267,29 +298,115 @@ def transcriptionStart_on_click(self):
         # Execute
         self.threadpool.start(worker)
 
+    @pyqtSlot()
+    def CheckConvert_SR_on_click(self):  
```

Comment:
I think all that should live somewhere in https://github.com/mozilla/DeepSpeech-examples/blob/r0.9/vad_transcriber/wavTranscriber.py as a frontend to `stt`